<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_class</class>
 <widget class="QWidget" name="add_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>地点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>考试时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>考试地点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>week：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_11">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_12">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_15">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_16">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>day(1-7)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学生(每行一个）：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>151</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
